### DISCUSSION TITLE
Constructing Wifi Packets

### DISCUSSION THREAD

**Discussion by dozan0** [dotpcap/packetnet#172] [Q&A]
There is no example of creating wifi packets. I am trying to send udp packet from one wifi adapter to another without connecting to access point. Can you help

**Comment by kayoub5** [comment]
@dozan0 Are you using WiFi monitor mode?

**Reply by dozan0** [reply]
Yes. My wifi adapters support monitor mode.

**Reply by PhyxionNL** [reply]
You also need a network driver that can inject packets.

**Reply by dozan0** [reply]
I can inject packets when wifi adapter connected to access point (not  monitor mode).  What kind of driver do i need . My wifi adapter's chipset is Atheros AR9271. So do i have to code my own driver or i can find it  somewhere? I can capture packets in monitor mode. Any other driver for inject in monitor mode?

**Reply by kayoub5** [reply]
What I do know is that you can inject normal Ethernet frames when adapter is NOT in monitor mode, I have not testing using monitor mode.
